qa/os/src/test/java/org/elasticsearch/packaging/test/PackagingTestCase.java

```diff
@@ -207,4 +185,87 @@ protected static Shell newShell() throws Exception {
         }
         return sh;
     }
+
+    /**
+     * Run the command to start Elasticsearch, but don't wait or test for success.
+     * This method is useful for testing failure conditions in startup. To await success,
+     * use {@link #startElasticsearch()}.
+     * @return Shell results of the startup command.
+     * @throws Exception when command fails immediately.
+     */
+    public Shell.Result runElasticsearchStartCommand() throws Exception {
+        switch (distribution.packaging) {
+            case TAR:
+            case ZIP:
+                return Archives.runElasticsearchStartCommand(installation, sh);
+            case DEB:
+            case RPM:
+                return Packages.runElasticsearchStartCommand(sh);
+            case DOCKER:
+                // nothing, "installing" docker image is running it
+        }
+        return Shell.NO_OP;
```

Comment:
Shouldn't this be an illegal state? Can we return this directly from the DOCKER case above and remove this bare return, so that if we added another packaging type we would not implicitly fall through to this?